### PR TITLE
Extension: add gatorRTC

### PR DESCRIPTION
**Pull request by martinwork**
@abchatra DO NOT MERGE
I think these problems should be fixed: https://github.com/sparkfun/pxt-gator-rtc/issues/3

### CODE REVIEW

**Comment by martinwork** [comment]
@loricrotser Thanks for the updates.
@abchatra I think this is good to go now.

**Comment by abchatra** [comment]
Blocks seems bit broken. Is this expected?
image

**Comment by loricrotser** [comment]
The blocks are initialized to 0 such that the user can set them as they see fit. Or are you referring to the multiple rows? I have not changed anything with this - to my knowledge it has always looked this way.

**Comment by abchatra** [comment]
I am referring to the **: , - -20** before the zero

**Comment by abchatra** [comment]
Oh this is a date format. It was hard to parse. Probably single line would be much better than multiple rows here.
